tests/std/tests/GH_000685_condition_variable_any/test.cpp

```diff
@@ -79,9 +82,136 @@ namespace {
         assert(m.num_locks() == 4);
 #endif // _HAS_CXX20
     }
+
+    // Minimal example inspired by LWG-4301, modified due to missing latch in C++11 to 20

```

Comment:
Somewhat confusing because we support C++14 as our minimal version, and `latch` was added in C++20.